### PR TITLE
fix(guide): fix expected solution for react-redux extract state logic challenge

### PR DESCRIPTION
**Pull request by ritute**
This challenge expects a different solution than the one in the guide.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [ ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/arabic/certifications/front-end-libraries/react-and-redux/extract-state-logic-to-redux/index.md [hunk 1]
```diff
@@ -10,26 +10,26 @@ localeTitle: استخراج الدولة المنطق إلى Redux
...
-  getState: () => store.state, 
-  dispatch: (action) => { 
-    if (action.type === ADD) { 
-      store.state = messageReducer(store.state, action); 
-    } 
-  } 
- }; 
-`
\ No newline at end of file
+ `const ADD = 'ADD';
```
Comment (thecodingaviator) [inline guide/arabic/certifications/front-end-libraries/react-and-redux/extract-state-logic-to-redux/index.md]:
```suggestion
 ```js
 const ADD = 'ADD';
```

File: guide/arabic/certifications/front-end-libraries/react-and-redux/extract-state-logic-to-redux/index.md [hunk 2]
```diff
@@ -10,26 +10,26 @@ localeTitle: استخراج الدولة المنطق إلى Redux
...
+         ...state,
+         action.message
+       ];
+     default:
+       return state;
+   }
+ };
+
+ const store = Redux.createStore(messageReducer); 
+`
```
Comment (thecodingaviator) [inline guide/arabic/certifications/front-end-libraries/react-and-redux/extract-state-logic-to-redux/index.md]: Make this three backticks

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Check comments

File: guide/arabic/certifications/front-end-libraries/react-and-redux/extract-state-logic-to-redux/index.md (hunk 2 above)
Comment (ritute) [reply guide/arabic/certifications/front-end-libraries/react-and-redux/extract-state-logic-to-redux/index.md]: I realized this, but I noticed the other guides I looked at in these languages didn't use the three backticks, so I left it like this for consistency and figured there might've been a reason for it.

**Comment by RandellDawson** [comment]
Solutions should include the entire solution (all code) and not just the parts needing to be changed.  The original versions did not have those either, but we now require full solutions, so that a user can copy and replace all code in the FCC editor and the solution will pass and show correctly on the site.

Please make sure you are adding full solutions.

**Comment by ritute** [comment]
@RandellDawson These are the full solutions that they can just copy + paste and will pass the test.

**Comment by RandellDawson** [comment]
@ritute When I look at the Challenge Seed, there is a lot more code that you are not including with your solution.  When I say "full", I mean all the original seed code plus the code needed to pass the challenge.

I only looked at the English version, but there are 96 lines of code in the seed and your "full" solution only has 22 lines in it.

Another thing here, is these should be put on separate PRs.  We expect you to check all the boxes on the PR and the 5th one states you have not put multiple languages on the same PR.

Why? Because we have translators who will validate you have not removed/added anything which is language dependent.  In this case, you also need to be removing the stub text for the each file and that would mean you are changing the text.  

Please only put the English version on this PR and split the others up into separate languages, **unless** the stubs are already removed on the other languages.

Thank you.

File: guide/arabic/certifications/front-end-libraries/react-and-redux/extract-state-logic-to-redux/index.md (hunk 2 above)
Comment (thecodingaviator) [reply guide/arabic/certifications/front-end-libraries/react-and-redux/extract-state-logic-to-redux/index.md]: Nope, this is fine. Please change it so it reflects the language too

**Comment by thecodingaviator** [comment]
@RandellDawson This solution passes for me on over [here](https://www.freecodecamp.rocks/learn/front-end-libraries/react-and-redux/extract-state-logic-to-redux) . Not sure what is happening

**Comment by RandellDawson** [comment]
@thecodingaviator You are correct.  Not sure what I was looking at before.  @ritute Just ignore my comments above about the full solution part.  However, you still need to remove the stub information from all of the articles.

**Comment by thecodingaviator** [comment]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**